### PR TITLE
The swift-backtrace library is no longer needed in 5.9

### PR DESCRIPTION
**Pull request by gwynne**
Per https://github.com/swift-server/swift-backtrace/pull/68.

### CODE REVIEW

**Comment by Joannis** [comment]
Isn't this API breaking for users moving to 5.9? If they rely on Backtrace themselves, through this package, this could lead to confusing situations.

**Comment by gwynne** [comment]
We (mercifully) don't have an `@_exported` line for this one, by some great virtue, so it's not part of Vapor's API. Relying on it as a transitive dependency if it's being used directly is incorrect usage.

File: Sources/Vapor/Error/StackTrace.swift [hunk 1]
```diff
@@ -34,12 +34,12 @@ public struct StackTrace: Sendable {
         return .init(rawFrames: .init(frames))
     }
 
-    #if os(Linux)
+    #if swift(<5.9) || os(Windows)
     private static let state = backtrace_create_state(CommandLine.arguments[0], /* supportThreading: */ 1, nil, nil)
     #endif
 
     static func captureRaw() -> [RawFrame] {
```
Comment (0xTim) [inline Sources/Vapor/Error/StackTrace.swift:41]: Do we need anything from this function _at all_ any more? I know we need to maintain the API, but `capture(skip:)` could just return `nil` in 5.9 which is non-breaking

**Review by 0xTim — APPROVED** [review]

**Comment by gwynne** [comment]
> Much nicer! But let's remove all the commented out code

Done 😉

**Review by 0xTim — APPROVED** [review]

**Comment by gwynne** [comment]
Bypassing the usual incorrectness of the API breakage checker.

**Pull request merged**